styles.css

```diff
@@ -0,0 +1,186 @@
+@import url("https://rsms.me/inter/inter.css");
+
+html {
+  font-family: "Inter", sans-serif;
+}
+
+@supports (font-variation-settings: normal) {
+  html {
+    font-family: "Inter var", sans-serif;
+  }
+}
+
+body {
+  max-width: max-content;
+  margin: auto;
+}
+
+.container {
+  width: 375px;
+  height: 6715px;
```

Comment:
- [OPTIONAL] Please make sure that the mobile version is responsive until the desktop screen size. :eyes: 
![image](https://user-images.githubusercontent.com/98363075/192096916-7200f494-6d01-4b23-ab7d-ee60b079e355.png)

Reply:
Thanks, ma'am so much.

Reply:
thanks, so much.